ops/pebble.py

```diff
@@ -2824,7 +2824,7 @@ def get_notices(
         type has nanosecond precision).
 
         Args:
-            users: Select which notices to return (instead of returning
+            users: Filter which users' notices to return (instead of returning
```

Comment:
I think filter is the wrong word here, as it's kind of doing the opposite -- "select=all" means it *does not* filter by user ID but returns notices for all users. So just a wording tweak:
```suggestion
            users: Change which users' notices to return (instead of returning
```